src/pages/en/upgrade-astro.mdx

```diff
@@ -0,0 +1,280 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Upgrade Guide
+description: How to upgrade your project to the latest version of Astro.
+i18nReady: false
+---
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+
+This guide will help you migrate from Astro v1 to Astro v2.
+
+Need to upgrade an older project to v1? See our [older migration guide](https://github.com/withastro/docs/blob/aa7e4f568f418657eba297778729a8aee2a36409/src/pages/en/migrate.mdx).
+
+You can update your project's version of Astro to the latest version using your package manager. If you're using Astro integrations, please also update those to the latest version.
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  # updates the astro dependency:
+  npm upgrade astro
```

Comment:
similar to how we recommend `npm create astro@latest` vs. just `npm create astro`, these package managers aren't great/consistent when it comes to caching. It's a good habit to say `astro@latest` in our docs just to be safe